src/main/java/be/acara/frontend/controller/UserController.java

```diff
@@ -129,4 +131,8 @@ public String likeOrDislikeEvent(@RequestParam("liked") boolean liked,
     public String getProfile(Principal principal) {
         return String.format("%s/%d", FORWARD_USER_DETAILS, userService.findByUsername(principal.getName()).getId());
     }
+
+    private void addLocalDateTime(Model model) {
+        model.addAttribute("now", LocalDateTime.now());
+    }
```

Comment:
I'm not sure whether replacing that one line with a seperate model is worth it, especially with readability in mind.